src/server/routes/annualData.js

```diff
@@ -69,7 +69,7 @@ function annualCurrentListData(req, res, next) {
       next();
     })
     .catch((error) => {
-      logger.error(`Status Code: ${error.statusCode}, Error Message: ${error.code}`);
+      logger.error(`Status Code: ${error.statusCode}, Error Message: ${error.code}, Endpoint: ${platformConfig.endpoints.annualLists[dataType]},Redirecting to: ${config.baseUrl}/404`);
```

Comment:
This `catch` actually will catch the errors both from the first call for the lists and the second call for the books. I am not sure how to do it, but it could be incorrect if we always return the same endpoint for all the errors.

Same as the other one in `src/server/routes/annualData.js` and the other two in `src/server/routes/monthData.js`

Reply:
Good point. It would make more sense to say which function was generating this error?